subsys/fs/pm.yml

```diff
@@ -5,3 +5,9 @@ nffs_storage:
   placement: {after: [mcuboot_storage, app]}
   size: CONFIG_PM_PARTITION_SIZE_NFFS_STORAGE
 #endif
+
+#ifdef CONFIG_FILE_SYSTEM_LITTLEFS
+littlefs:
+  placement: {after: [mcuboot_storage, app]}
```

Comment:
```suggestion
  placement: {after: [nffs_storage, mcuboot_storage, app]}
```

Reply:
It's a bit weird to use both file-systems but I guess it has a use-case.

Reply:
not any more.